build-system/sauce-credentials.json

```diff
@@ -4,6 +4,10 @@
     "rsimha@ampproject.org": {
       "username": "rsimha",
       "access_key_encoded": "MDg2ZjY4MzQtMWRkZi00YTUyLTgwMWUtMmUwODIyNDY2OTc0Cg=="
+    },
+    "cramforce@users.noreply.github.com": {
+      "username": "amphtml-malte",
+      "access_key_encoded": "Y2Q2ODY1ZTQtZGEwYi00OGEzLWFjZTMtYWMzMDUyMzg2ZjRj"
```

Comment:
I noticed that encoding the token using the js `btoa` lib does not add a `==` to the end of the token, while using the command line tool from npm does. Shouldn't make a difference to the end result though.